### PR TITLE
(#55) [Fixed]: Environment variable names

### LINKED ISSUE #55: Improve environment variables in settings
**Is your feature request related to a problem? Please describe.**
At the moment we have three environment variables:

* DEBUG
* APF_ELASTIC_HOST_NAME
* APF_ELASTIC_HOST_PORT

It haves different pattern names. Also, the elastic search variables does not use `environ`.

**Describe the solution you'd like**
* Change DEBUG variable name to APF_DEBUG
* Change ES variables for using `environ`
* Crear `env.sample`file
* Update documentation notifying DEBUG needs to be True in development environment (for statics serve)

**Describe alternatives you've considered**
No

**Additional context**
No needed

### PR DESCRIPTION
**Pull request by francescarpi**
## Description

Fix #55 

* I unified some env var names.
* I added the `.env.dev.sample`

## Code formating

The code has to be well formatted following the formatting rules. For example in Python the code has to follow the PEP8.

### CODE REVIEW

File: jobs/settings/.env.dev.sample [hunk 1]
```diff
@@ -0,0 +1,3 @@
+; To use dev env ini file.
```
Comment (jbagot) [inline jobs/settings/.env.dev.sample:1]: Normally the .env file and the .env.dev.sample are in the root of the project and then in the settings you can specify the path of the .env file in the readEnv method.

File: jobs/settings/__init__.py [hunk 2]
```diff
@@ -3,7 +3,9 @@
 import environ
 
 env = environ.Env(
-    DEBUG=(bool, False)
+    DEBUG=(bool, False),
+    ELASTIC_HOST_NAME=(str, 'localhost'),
```
Comment (jbagot) [inline jobs/settings/__init__.py:7]: this is completely fine, but I normally don't use it like this. Normally I have the Env() method empty and then I add the casting in each var.
For example
`ELASTIC_HOST_PORT = env.str("ELASTIC_HOST_PORT", default="9200")`, but up to you.

Comment (francescarpi) [reply jobs/settings/__init__.py:7]: Oh, nice! I'm going to change it

File: jobs/settings/.env.dev.sample (hunk 1 above)
Comment (francescarpi) [reply jobs/settings/.env.dev.sample:1]: Interesting. I'm going to do it.
Thx

Comment (francescarpi) [reply jobs/settings/.env.dev.sample]: Do it! Can you check it again @jbagot , pls?

File: jobs/settings/__init__.py (hunk 2 above)
Comment (francescarpi) [reply jobs/settings/__init__.py]: Done!

**Review by jbagot — APPROVED** [review]
Yes, I like it now, thanks :+1:

**Pull request merged**